b2share/modules/schemas/root_schemas/root_schema_v0.json

```diff
@@ -5,132 +5,191 @@
                         }
                     }
-                }
-            },
-            "community": {
-                "title": "Community",
-                "description": "The community to which the record has been submitted.",
-                "type": "string"
+                },
+                "uniqueItems": true
             },
-            "title": {
+            "titles": {
                 "title": "Title",
-                "description": "The title of the uploaded resource: a name that indicates its content.",
-                "type": "string"
-            },
-            "description": {
-                "title": "Description",
-                "description": "A more elaborate description of the resource. Focus on a content description that makes it easy for others to find, and to interpret its relevance.",
-                "type": "string"
-            },
-            "creators": {
-                "title": "Authors",
-                "description": "The record author(s).",
+                "description": "The title(s) of the uploaded resource, or a name by which the resource is known ",
                 "type": "array",
-                "items": {"type": "string"},
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "title": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "minItems":1,
                 "uniqueItems": true
             },
-            "open_access": {
-                "title": "Open Access",
-                "description": "Indicate whether the record's files are publicly accessible or not. In case of restricted access the uploaded files will only be accessible by the record's owner and the community administrators. Please note that the record's metadata is always publicly accessible. ",
-                "type": "boolean"
-            },
-            "embargo_date": {
-                "title": "Embargo Date",
-                "description": "The date marking the end of the embargo period. The record will be marked as open access on the specified date at midnight. Please note that the record metadata is always publicly accessible, and only the data files can have private access.",
+            "publisher": {
+                "title": "Publisher",
+                "description": "The entity responsible for making the resource available, either a person, an organization, or a service.",
                 "type": "string",
-                "format": "date-time"
+                "default": "https://b2share.eudat.eu"
             },
-            "licence": {
-                "title": "Licence",
-                "description": "Specify the license under which this data set is available to the users (e.g. GPL, Apache v2 or Commercial). Please use the License Selector for help and additional information.",
+            "publication_year": {
```

Comment:
@lnielsen thanks.
@emanueldima so I would suggest to have a "publication_date" field with a string type and only accept "ISO8601" for now. We can later accept EDFT. The change would only imply documentation change and validation change in the backend, without changing the JSON schema.
When exporting to datacite we would extract a "publication year" out of this "publication_date".